doc/modules/model_evaluation.rst

```diff
@@ -1936,6 +1937,37 @@ function::
   >>> mean_squared_log_error(y_true, y_pred)
   0.044...
 
+.. _mean_absolute_percentage_error:
+
+Mean absolute percentage error
+------------------------------
+The :func:`mean_absolute_percentage_error` is particularly interesting because
+it is directly understood by the users who don't have technical background.
+It is modification of `mean_absolute_error`. The loss is calculated by taking
+average of all differences between the target and the prediction divided by
+the target and multiplying resulting term by 100.0 to convert it into percentage.
```

Comment:
explain what it does : "penalize errors in percentage" vs absolute value for MAE. It is therefore blind to global scaling of the target as long as the prediction is scaled like the true y. Try to rephrase or just copy paste
from some wikipedia page or similar.

Reply:
changed